### DISCUSSION TITLE
when I try to install on to ubuntu 18.04 I get repository does not have a release file. Version 18.04.05

### DISCUSSION THREAD

**Discussion by Kat35601** [Security-Onion-Solutions/securityonion#2786] [General] [answered]
when I try to install on to ubuntu 18.04 I get repository does not have a release file. Version 18.04.05

ERROR ] Command '['apt-get', '-q', 'update']' failed with return code: 100
[ERROR ] stdout: Hit:1 http://repo.saltstack.com/py3/ubuntu/18.04/amd64/archive/3002.2 bionic InRelease
[ERROR ] stderr: E: The repository 'http://ppa.launchpad.net/securityonion/stable/ubuntu bionic Release' does not have a Release file.
[ERROR ] retcode: 100
[ERROR ] An error was encountered while checking the newest available version of package(s): E: The repository 'http://ppa.launchpad.net/securityonion/stable/ubuntu bionic Release' does not have a Release file.
[ERROR ] Command '['apt-get', '-q', 'update']' failed with return code: 100
[ERROR ] stdout: Hit:1 http://us.archive.ubuntu.com/ubuntu bionic InRelease
[ERROR ] stderr: E: The repository 'http://ppa.launchpad.net/securityonion/stable/ubuntu bionic Release' does not have a Release file.
[ERROR ] retcode: 100
[ERROR ] An error was encountered while installing package(s): E: The repository 'http://ppa.launchpad.net/securityonion/stable/ubuntu bionic Release' does not have a Release file.
[ERROR ] Command '['apt-get', '-q', 'update']' failed with return code: 100
[ERROR ] stdout: Hit:1 http://repo.saltstack.com/py3/ubuntu/18.04/amd64/archive/3002.2 bionic InRelease
[ERROR ] stderr: E: The repository 'http://ppa.launchpad.net/securityonion/stable/ubuntu bionic Release' does not have a Release file.
[ERROR ] retcode: 100
[ERROR ] An error was encountered while installing package(s): E: The repository 'http://ppa.launchpad.net/securityonion/stable/ubuntu bionic Release' does not have a Release file.
[ERROR ] Command '['apt-get', '-q', 'update']' failed with return code: 100
[ERROR ] stdout: Hit:1 http://repo.saltstack.com/py3/ubuntu/18.04/amd64/archive/3002.2 bionic InRelease
[ERROR ] stderr: E: The repository 'http://ppa.launchpad.net/securityonion/stable/ubuntu bionic Release' does not have a Release file.
[ERROR ] retcode: 100
[ERROR ] An error was encountered while installing package(s): E: The repository 'http://ppa.launchpad.net/securityonion/stable/ubuntu bionic Release' does not have a Release file.

**Comment by weslambert** [comment]
Could you please provide the full set of commands/operations that led you to this issue?

**Reply by Kat35601** [reply]
git clone https://github.com/Security-Onion-Solutions/securityonion
cd securityonion
sudo bash so-setup-network

**Comment by weslambert** [comment, marked as answer]
This looks like an issue with the version of Security Onion you are attempting to run.  This forum is for Security Onion 2.  It looks like your issue is related to Security Onion 16.04 (https://github.com/Security-Onion-Solutions/security-onion/issues/1808).  If you have an issue with regard to Security Onion 2, please open a new discussion.

**Comment by lruppert** [comment]
#4486 is a similar issue with 2.3.52 and Ubuntu 18.04
